src/workloads/execution/BackgroundIndexConstruction.yml

```diff
@@ -0,0 +1,156 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the write performance impact of creating indexes on a very large collection.
+  It synthetically lowers the number of concurrent operations allowed to represent a saturated
+  server.
+
+Keywords:
+- stress
+- indexes
+- InsertRemove
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 1024
+      socketTimeoutMS: 3600000  # = 1 hour
+
+Actors:
+# Phase 1: Insert enough data to ensure than an index build on each field spills to disk with a
+# memory limit of 100MB.
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000000
+    BatchSize: 1000
+    Document:
+      randomInt: {^RandomInt: {min: -10000000, max: 10000000}}
+      randomString: {^RandomString: {length: 16}}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Lower the memory limit for spilling to disk so that it occurs more often.
+- Name: Setup
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    # Fsync to force a checkpoint and quiesce the system.
```

Comment:
We have a quiesce actor you could use instead of the fsync, although it's documentation isn't as reassuring as I would like: https://github.com/mongodb/genny/blob/master/src/workloads/docs/QuiesceActor.yml It does an fsync, but it does more than that also. 

https://github.com/mongodb/genny/blob/d4897cf792d3ea30c40c94c5e04ce715a918cd1f/src/gennylib/src/quiesce.cpp